### PR TITLE
Fix issues in RegisterBtcTransaction class gas calculation

### PR DESCRIPTION
**Pull request by pamgonzalez**

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/performance/RegisterBtcTransactionTest.java [hunk 1]
```diff
@@ -142,6 +142,9 @@ private BridgeStorageProviderInitializer generateInitializerForLock(int minBtcBl
 
             Helper.generateAndAddBlocks(btcBlockChain, numberOfLockConfirmations);
 
+            thisRepository.commit();
+            thisRepository.commit();
```
Comment (josedahlquist) [inline rskj-core/src/test/java/co/rsk/peg/performance/RegisterBtcTransactionTest.java:146]:
```suggestion
            thisRepository.commit();
```

File: rskj-core/src/test/java/co/rsk/peg/performance/RegisterBtcTransactionTest.java [hunk 2]
```diff
@@ -110,8 +110,8 @@ private BridgeStorageProviderInitializer generateInitializerForLock(int minBtcBl
             // Sender and amounts
             BtcECKey from = new BtcECKey();
             Address fromAddress = from.toAddress(networkParameters);
-            Coin fromAmount = Coin.CENT.multiply(Helper.randomInRange(10, 100));
-            Coin lockAmount = fromAmount.divide(Helper.randomInRange(2, 10));
+            Coin fromAmount = Coin.CENT.multiply(Helper.randomInRange(100, 150));
+            Coin lockAmount = fromAmount.divide(Helper.randomInRange(2, 4));
```
Comment (josedahlquist) [inline rskj-core/src/test/java/co/rsk/peg/performance/RegisterBtcTransactionTest.java:114]: Is this change required to fix the test?

**Review by josedahlquist — CHANGES_REQUESTED** [review]

File: rskj-core/src/test/java/co/rsk/peg/performance/RegisterBtcTransactionTest.java (hunk 2 above)
Comment (pamgonzalez) [reply rskj-core/src/test/java/co/rsk/peg/performance/RegisterBtcTransactionTest.java:114]: Yes, with the old values, the test didn't work.

**Review by josedahlquist — DISMISSED** [review]

**Comment by pamgonzalez** [comment]
pipeline:run

**Review by aeidelman — DISMISSED** [review]

**Comment by pamgonzalez** [comment]
pipeline:run

**Review by marcos-iov — APPROVED** [review]

**Comment by pamgonzalez** [comment]
pipeline:run

**Comment by alejandrod-iov** [comment]
pipeline:run

**Review by aeidelman — APPROVED** [review]
